doc/nrf/samples.rst

```diff
@@ -11,6 +11,8 @@ Zephyr also provides a variety of :ref:`zephyr:samples-and-demos`, including ver
 These samples are a good starting point for understanding how to put together your own application.
 However, Zephyr samples and applications are not tested and verified to work with the |NCS| releases.
 
+.. samples_general_info_start
+
 General information about samples in the |NCS|
```

Comment:
Was the missing space between intro sentence and the first bullet point intentional?
It doesn't render that bad in this case, but we do state that the first bullet point line should have a blank line before it.

Reply:
It is intentional to have this kind of formatting, like <dt><dd> in XML:
![image](https://github.com/user-attachments/assets/6e128e02-51d9-4066-9a4f-ca43718cb317)